cpp/src/Ice/AddDefaultPluginFactories_all.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) ZeroC, Inc.
+
+#include "AddDefaultPluginFactories.h"
+
+#include <algorithm>
+
+using namespace std;
+
+extern "C"
+{
+    Ice::Plugin* createIceUDP(const Ice::CommunicatorPtr&, const std::string&, const Ice::StringSeq&);
+    Ice::Plugin* createIceTCP(const Ice::CommunicatorPtr&, const std::string&, const Ice::StringSeq&);
+    Ice::Plugin* createIceWS(const Ice::CommunicatorPtr&, const std::string&, const Ice::StringSeq&);
+    Ice::Plugin* createIceSSL(const Ice::CommunicatorPtr&, const std::string&, const Ice::StringSeq&);
+}
+
+void
+IceInternal::addDefaultPluginFactories(vector<Ice::PluginFactory>& pluginFactories)
+{
+    // These built-in plug-ins are always available and not returned by a Ice::namePluginFactory function.
+    vector<Ice::PluginFactory> defaultPluginFactories{{"IceTCP", createIceTCP}, {"IceSSL", createIceSSL}};
+
+    if (find_if(
```

Comment:
What about replacing with none_of

https://en.cppreference.com/w/cpp/algorithm/all_any_none_of

Reply:
Fixed as suggested. There are a few other places where we should replace find_if.